src/web/csp.rs

```diff
@@ -54,7 +54,7 @@ impl Csp {
         // the MIME type to allow loading favicons.
         //
         // Images from other HTTPS origins are also temporary allowed until issue #66 is fixed.
-        result.push_str("; img-src 'self' https:");
+        result.push_str("; img-src 'self' https: data:");
```

Comment:
This change allows to have inlined SVG in the CSS.

Reply:
If it's an issue, we can instead use URLs and have the SVG images stored in files.